prow/jobs/kyma/tools/failery/failery.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  run_if_changed: "^tools/failery/"
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+        securityContext:
+          privileged: true
```

Comment:
If this job will not require DIND then it should work as a normal user, not privileged.